java/connector-framework-internal/src/main/java/org/identityconnectors/framework/impl/serializer/CommonObjectHandlers.java

```diff
@@ -657,58 +634,57 @@ public void serialize(final Object object, final ObjectEncoder encoder) {
 
         HANDLERS.add(
 
-        new AbstractObjectSerializationHandler(OperationOptions.class, "OperationOptions") {
+                new AbstractObjectSerializationHandler(OperationOptions.class, "OperationOptions") {

```

Comment:
Please avoid such formatting-only changes.

Reply:
@matusmacik this is still in place (also several other occurences in the same file)

Reply:
Seems that the change was somehow ignored, I switched to a different editor and now I see the diff should be ok, Ill follow up with a commit.